camera_id: 4              # camera id
max_speed: 1.0            # m/s
max_turn: 0.8             # rad/s
tgt_stop_dist: 1.0        # m, stop distance threshold for target to follow, default is 1.0 m
obs_stop_dist: 0.5        # m, stop distance threshold for nearby obstacles, default is 0.5 m
compute_period: 25        # ms, (jetson max realsense fps is ~40 fps = 25 ms) --> for color detection set to 33 ms
use_aruco: true   
camera_fps: 60